ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -615,25 +615,25 @@ device_classes: !ZenPackSpec
                 aliases: {if_sent_packets__persec: null}
             meter: network.outgoing.packets.rate
         graphs:
-          Network Packet Rate:
```

Comment:
Does zenpacklib migrate graphs itself? E.g. - it looks that you create a new grph ids.